config/locales/en.yml

```diff
@@ -0,0 +1,33 @@
+# Files in the config/locales directory are used for internationalization
```

Comment:
Eventually we should do something like this, but I don't think we need it in this iteration of the app, and there are a couple of new ways I've been learning about doing this other than a yml file which seem pretty interesting.... we can put it in the icebox and discuss it later.